app/code/community/Bolt/Boltpay/etc/system.xml

```diff
@@ -355,11 +355,39 @@ This function is called when the Bolt checkout modal is closed.
               <show_in_store>0</show_in_store>
               <sort_order>106</sort_order>
             </close>
+            <add_button_to_minicart translate="label">
+              <label>Add Bolt Button to the MiniCart Popup</label>
```

Comment:
I'm wondering whether we should name this in more generic way. For instance argento theme has popup with checkout button. It's not minicart per se. 

The way I see is we need an option to decide whether we only inject connect.js to cart page or we inject it everywhere